### PR TITLE
Spellcheck

### PR DESCRIPTION
**Pull request by facelessuser**
Move to pyspelling and check settings, source, and documents.

### CODE REVIEW

File: bh_core.sublime-settings [hunk 1]
```diff
@@ -367,18 +367,15 @@
         // Angle
         {
             "name": "angle",
-            "open": "(<)(?=[^?%]|$)",
```
Comment (facelessuser) [inline bh_core.sublime-settings]: This wasn't meant to be included here.  Revert this change to angle matching.

File: bh_core.sublime-settings [hunk 2]
```diff
@@ -86,16 +86,16 @@
     // Experimental: Special matching mode for block cursor.
     // Essentially, this provides a matching mode that makes a little more
     // sense to some in regards to the visual representation of block cursors.
-    // This will ignore "bracket_outside_adjacent".
+    // This will ignore `bracket_outside_adjacent`.
     "block_cursor_mode": false,
 
-    // When "bracket_outside_adjacet" is set, and a plugin command explicitly sets
-    // "no_outside_adj" to "None" instead of "true" or the default "false",
+    // When `bracket_outside_adjacet` is set, and a plugin command explicitly sets
```
Comment (michaelblyons) [inline bh_core.sublime-settings]: Should be `bracket_outside_adjacent` (missing `n`).

Comment (facelessuser) [reply bh_core.sublime-settings]: Thanks for pointing this out.  I had put this on ice for a bit as I wasn't happy with my first implementation of the spelling tool and only recently got around to refactoring.  This got overlooked as I'm specifically avoiding things in back ticks (mainly non-words like function names and settings values etc.).
